public/walkthroughs/1/walkthrough.adoc

```diff
@@ -42,6 +42,15 @@ This walkthrough describes how  to create such an integration, using OpenShift,
 // end::master-1[]
 
 
+[type=walkthroughResource,serviceName=fuse]
```

Comment:
@pb82  how do i add an AMQ resource?
also, how do i know what serviceNames are avail

Reply:
@finp this works for all default middleware services which you can find in this list: https://github.com/integr8ly/tutorial-web-app/blob/master/src/common/serviceInstanceHelpers.js#L85